### PR TITLE
The Great Combinator Renaming

### PR DESCRIPTION
**Pull request by littleredcomputer**

### CODE REVIEW

File: docs/genjax/library/generative_functions/combinators/unfold.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 GenJAX's `UnfoldCombinator` is a combinator which implements a scan-like pattern of control flow by utilizing `jax.lax.scan`.
 
-::: genjax.generative_functions.combinators.Unfold
+::: genjax.generative_functions.combinators.unfold_combinator
     options:
       show_root_heading: true
-      members:
```
Comment (sritchie) [inline docs/genjax/library/generative_functions/combinators/unfold.md:8]: nice catch!

File: docs/genjax/library/generative_functions/static.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ Up front, here's a representative program, (with syntactic sugar, using the cons
...
 console = genjax.console()
 
-@Static
+@static
 def beta_bernoulli_process(u):
     p = beta(0.0, u) @ "p"
     v = bernoulli(p) @ "v" # sweet
     return v
 
-print(console.render(beta_bernoulli_process))
```
Comment (sritchie) [inline docs/genjax/library/generative_functions/static.md:27]: oh nice, does this work without the print?

File: notebooks/intermediate/smc/smc_with_translators.ipynb [hunk 3]
```diff
@@ -181,7 +181,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.9"
```
Comment (sritchie) [inline notebooks/intermediate/smc/smc_with_translators.ipynb:184]: funny that this doesn't get stripped as well...

Comment (sritchie) [reply notebooks/intermediate/smc/smc_with_translators.ipynb:184]: I will upgrade so I don't revert this!

**Review by sritchie — APPROVED** [review]

File: tests/generative_functions/test_drop_arguments.py [hunk 4]
```diff
@@ -21,9 +21,9 @@
 
 class TestDropArguments:
     def test_drop_arguments_as_kernel_in_map(self):
-        @genjax.Map(in_axes=(0,))
+        @genjax.map_combinator(in_axes=(0,))
         @genjax.DropArguments
```
Comment (horizon-blue) [inline tests/generative_functions/test_drop_arguments.py:25]: This is unrelated to the renaming in this PR but I wonder if we could use [the lower-case variant of `drop_arguments` here](https://github.com/probcomp/genjax/blob/e26fcd3321c07aece4a5191390f9ac7a3bf13bf1/src/genjax/_src/generative_functions/drop_arguments.py#L186)?

**Review by horizon-blue — COMMENTED** [review]
Thank you for making the changes @littleredcomputer! :D

**Pull request merged**